__init__.py

```diff
@@ -51,6 +51,51 @@ class PairingMode(str, Enum):
     GUI = "gui"  # gui - click buttons
 
 
+
+class GUIHelper:
+    """ helper class for better handling of specific setup GUI like list"""
+
+    def __init__(self) -> None:
+        pass
+
+    def distinguish_models_tts(self, model):
+        """ distinguish the model to a separate list for engines and voices
+        to build a nested list for the GUI """
+        engines_model = []
+
+        for entry in model:
+            if entry["engine"] not in [engine["engine"] for engine in engines_model]:
+                engines_model.append({"engine": entry["engine"], "plugin_name": entry["plugin_name"], "supports_male_voice": False, "supports_female_voice": False, "supports_offline_mode": False, "supports_online_mode": False,  "voiceModel": []})
+
+        for entry in model:
+            for engine in engines_model:
+                if entry["engine"] == engine["engine"]:
+                    engine["voiceModel"].append(entry)
```

Comment:
nitpick, in python we dont use CamelCase, change to voice_model for consistency across all code base and projects

Reply:
There is no rule for this nitpick and it seems quite like forcing a coding style